posts/Mincong/2016-08-22-new-implementation-of-mass-indexer-using-jsr-352.adoc

```diff
@@ -138,13 +138,14 @@ massIndexer = massIndexer.checkpointFreq( 1000 );
 == Run
 
 For further usage, please check my GitHub repo
-https://github.com/mincong-h/gsoc-hsearch[gsoc-hsearch]. If you want to play with it,
-you can download the code and build it with Maven: 
+https://github.com/mincong-h/gsoc-hsearch/tree/1.0[gsoc-hsearch]. If you want
+to play with it, you can download the code and build it with Maven: 
 
 [source]
 ----
 $ git clone -b 1.0 git://github.com/mincong-h/gsoc-hsearch.git
 $ cd gsoc-hsearch
+$ git checkout -b 1.0 origin/1.0
```

Comment:
I had changed the clone command above already to checkout the 1.0 branch, so there is no need for that :)